test/js/components/epics/detail.test.jsx

```diff
@@ -445,6 +449,39 @@ describe('<EpicDetail/>', () => {
     });
   });
 
+  describe('fetching more tasks', () => {
+    test('fetches next page of tasks', async () => {
+      const { findByText, getByText } = setup({
+        initialState: {
+          ...defaultState,
+          tasks: {
+            p1: {
+              ...defaultState.tasks.p1,
+              next: {
+                epic1: 'next-task-url',
+              },
+              count: {
+                epic1: defaultState.tasks.p1.tasks.length + 5,
+              },
+            },
+          },
+        },
+      });
+
+      const btn = getByText('Load More');
+      fireEvent.click(btn);
+
+      expect(fetchObjects).toHaveBeenCalledWith({
+        filters: { project: 'p1', epic: 'epic1' },
```

Comment:
I added this test to cover the case where we're fetching a next page of epic-level tasks.